packages/react-instantsearch-hooks-dom/src/ui/__tests__/RefinementList.test.tsx

```diff
@@ -0,0 +1,604 @@
+                        >
+                          <animatetransform
+                            attributeName="transform"
+                            dur="1s"
+                            from="0 18 18"
+                            repeatCount="indefinite"
+                            to="360 18 18"
+                            type="rotate"
+                          />
+                        </path>
+                      </g>
+                    </g>
+                  </svg>
+                </span>
+              </form>
+            </div>
+          </div>
+          <ul
+            class="ais-RefinementList-list"
+          >
+            <li
+              class="ais-RefinementList-item ais-RefinementList-item--selected"
+            >
+              <label
+                class="ais-RefinementList-label"
+              >
+                <input
+                  checked=""
+                  class="ais-RefinementList-checkbox"
+                  type="checkbox"
+                  value="Insignia™"
+                />
+                <span
+                  class="ais-RefinementList-labelText"
+                >
+                  Insignia™
+                </span>
+                <span
+                  class="ais-RefinementList-count"
+                >
+                  746
+                </span>
+              </label>
+            </li>
+            <li
+              class="ais-RefinementList-item"
+            >
+              <label
+                class="ais-RefinementList-label"
+              >
+                <input
+                  class="ais-RefinementList-checkbox"
+                  type="checkbox"
+                  value="Samsung"
+                />
+                <span
+                  class="ais-RefinementList-labelText"
+                >
+                  Samsung
+                </span>
+                <span
+                  class="ais-RefinementList-count"
+                >
+                  633
+                </span>
+              </label>
+            </li>
+          </ul>
+        </div>
+      </div>
+    `);
+  });
+
+  test('displays a no results fallback', () => {
+    const props = createProps({ noResults: 'No results' });
+    const { container } = render(<RefinementList {...props} />);
+
+    expect(container).toMatchInlineSnapshot(`
+      <div>
+        <div
+          class="ais-RefinementList"
+        >
+          <div
+            class="ais-RefinementList-noResults"
+          >
+            No results
+          </div>
+        </div>
+      </div>
+    `);
+  });
+
+  test('forwards a custom class name to the root element', () => {
+    const props = createProps({ className: 'MyRefinementList' });
+    const { container } = render(<RefinementList {...props} />);
+
+    expect(document.querySelector('.ais-RefinementList')).toHaveClass(
+      'MyRefinementList'
+    );
+    expect(container).toMatchInlineSnapshot(`
```

Comment:
I don't think we need this snapshot.

Reply:
Addressed in [445448c](https://github.com/algolia/react-instantsearch/pull/3381/commits/445448cb0cadfd537e1e71a6f7074db1e9789439)